spec/index.html

```diff
@@ -1243,42 +1252,31 @@ <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
 
 </section>
 
-<section id="section-Acknowledgments" class="informative">
+<section id="section-Acknowledgments" class="informative appendix">
   <h2>Acknowledgments</h2>
 
-  <p>The editors acknowledge valuable contributions from Thomas Baker,
-    Tim Berners-Lee, David Booth, Dan Brickley, Gavin Carothers, Jeremy Carroll,
-    Pierre-Antoine Champin, Dan Connolly, John Cowan, Martin J. Dürst,
-    Alex Hall, Steve Harris, Sandro Hawke, Pat Hayes, Ivan Herman, Peter F. Patel-Schneider,
-    Addison Phillips, Eric Prud'hommeaux, Nathan Rixham, Andy Seaborne, Leif Halvard Silli,
-    Guus Schreiber, Dominik Tomaszuk, and Antoine Zimmermann.</p>
-
-  <p>The membership of the RDF Working Group included Thomas Baker,
-    Scott Bauer, Dan Brickley, Gavin Carothers, Pierre-Antoine Champin,
-    Olivier Corby, Richard Cyganiak, Souripriya Das, Ian Davis, Lee Feigenbaum,
-    Fabien Gandon, Charles Greer, Alex Hall, Steve Harris, Sandro Hawke,
-    Pat Hayes, Ivan Herman, Nicholas Humfrey, Kingsley Idehen, Gregg Kellogg,
-    Markus Lanthaler, Arnaud Le Hors, Peter F. Patel-Schneider,
-    Eric Prud'hommeaux, Yves Raimond, Nathan Rixham, Guus Schreiber,
-    Andy Seaborne, Manu Sporny, Thomas Steiner, Ted Thibodeau, Mischa Tuffield,
-    William Waites, Jan Wielemaker, David Wood, Zhe Wu, and Antoine Zimmermann.</p>
-</section>
-
-<section class="appendix informative" id="changes">
-  <h2>Changes between RDF 1.0 and RDF 1.1</h2>
+  <p>The following people have contributed to this specification:
+    <span id="gh-contributors"></span>
```

Comment:
A number of the previously listed people will not be listed anymore... I think that's a shame...

Reply:
Apart from that, that's a super cool feature of Respec hat I didn't know about ;-)

Reply:
We certainly need to figure out how to deal with 1.1 contributors, in addition to 1.0. A reference to the contributors section in the previous spec(s) would be one way, or multiple contributor sections. Ultimately, this could prove to be difficult to manage. But, I think it’s a group decision on how to recognize previous contributions. Note that we do maintain a list of Former Editors separately.

I think @iherman had a script to get the current WG membership and create a list for inclusion automatically. 

similarly, what level of the “previous changes” do we try to maintain? Some have a lot of transition detail (WD, CR, PR, REC) that isn’t as pertinent now.

Reply:
@gkellogg I think what you referred to is here:

https://github.com/w3c/publ_ack

I have not looked at the script for a while, because recent groups that I worked with had their own way of creating the list of acknowledgements, but I presume it is still operational...

Reply:
Just came up with this using the W3C API: https://github.com/w3c/rdf-common/pull/3. (With an assist from ChatGPT).

Reply:
Yep, that is what my script uses as well!